### ISSUE TITLE
Norwegian characters trick highlighting

### ISSUE THREAD

**Issue by Osse** [easymotion/vim-easymotion#28]
If a word in the text starts with æ, ø or å the highlighting is disturbed from the next word on. The highlighted character is the second or third character in the word. Some words aren't highlighted at all. 

See the following picture. The last sentence of the second paragraph contains some Norwegian gibberish.
![Without EasyMotion](http://folk.ntnu.no/oystewa/testcase.png)

In the following picture `<Leader>w` is started.
![Without EasyMotion](http://folk.ntnu.no/oystewa/testcase2.png)
From the first word after "øl", the highlighting is incorrect. E.g. the 'g' is highlighted instead of the 'v' in 'og'. 'øser' isn't highlighted at all, nor is is the word changed to show the target key. But if I press 'w', which is the correct target key, the cursor moves to the first character of the word. 

After 'øser', 'og' and 'ærlige' are correctly changed to show the target key but the highlighting is wrong. On the word 'mennesker' the target key is put on the second character instead of the first and the highlighting is moved to the third character.  Again, pressing 'z' takes me to the first character of the word. From the next line on, everything is correct.

Long story short: The movements are always correct; the highlighting is disturbed, but only is the character _starts_ with æ, ø or å. I am guessing there is a regex somewhere that messes things up. I'm not very good with Vimscript. I'll try to take a look at it when I have the time.

PS: The same bug is also induced with also happens with ä, ö, é and è.
